figures/vis/visualization-datasource-smoketest-sequence.mmd

```diff
@@ -0,0 +1,14 @@
+sequenceDiagram
+    box Cypress Test
+    actor e2e as Mock User
+    participant Browser
+    participant ott-vis-datasource
+    end
+    
+    e2e->>+Browser : mock user visits
+    Browser->>-ott-vis-datasource : requests data
+    ott-vis-datasource->>+ott-collector : makes outbound request
+    ott-collector-->>ott-vis-datasource : returns data
```

Comment:
In this test scenario, the request should never actually reach ott-collector. It shouldn't go outside the test environment because it will get intercepted by cypress.